eisbuk-admin/src/components/atoms/CustomerCard/ActionButtons.tsx

```diff
@@ -70,27 +78,76 @@ const ActionButtons: React.FC<ActionButtonProps> = ({
         startIcon={<Mail />}
         className={classes.actionButton}
         color="primary"
-        onClick={() => setSendMailDialog(true)}
+        onClick={() => setSendBookingsMethod(SendBookingLinkMethod.Email)}
         data-testid={__sendBookingsEmailId__}
         variant="contained"
         // disable button if email or secret key not provided
         disabled={!(customer?.email && customer?.secretKey)}
       >
         {t(ActionButton.SendBookingsEmail)}
       </Button>
+      <Button
+        startIcon={<Mail />}
+        className={classes.actionButton}
+        color="primary"
+        onClick={() => setSendBookingsMethod(SendBookingLinkMethod.SMS)}
+        data-testid={__sendBookingsSMSId__}
+        variant="contained"
+        // disable button if phone number or secret key not provided
+        disabled={!(customer?.phone && customer?.secretKey)}
+      >
+        {t(ActionButton.SendBookingsSMS)}
+      </Button>
 
       <ConfirmDialog
-        open={sendMailDialog}
-        title={t(Prompt.SendEmailTitle)}
-        setOpen={setSendMailDialog}
-        onConfirm={sendBookingsEmail}
+        open={Boolean(sendBookingsMethod)}
+        title={promptTitle}
+        setOpen={(open: boolean) => (open ? null : setSendBookingsMethod(null))}
+        onConfirm={() => handleSendBookingsLink(sendBookingsMethod!)}
       >
-        {sendMailPromptMessage}
+        {promptBody}
       </ConfirmDialog>
     </div>
   );
 };
 
+interface GetDialogPrompt {
+  (payload: {
+    method: SendBookingLinkMethod | null;
+    email?: string;
+    phone?: string;
+  }): {
+    promptTitle: string;
+    promptBody: string;
+  };
+}
+
+/**
+ * Gets prompt text (title and body) for dialog prompt based on passed method.
+ * For type safety, it accepts `null` as `method` (and returns empty strings for `promptTitle` and `promptBody`),
+ * but that should not happen in reality.
+ * @param {string | null} param.method "email" or "sms"
+ * @param {string} param.email (optional) customer's email
+ * @param {string} param.phone (optional) customer's phone
+ * @returns
+ */
+const getDialogPrompt: GetDialogPrompt = ({ method, email, phone }) => {
+  if (!method) return { promptTitle: "", promptBody: "" };
+
+  const promptLookup = {
+    [SendBookingLinkMethod.Email]: {
+      promptTitle: i18n.t(Prompt.SendEmailTitle),
+      promptBody: [i18n.t(Prompt.ConfirmEmail), email, "?"].join(" "),
```

Comment:
After changing the translation string this should be changed to `i18n.t(Prompt.ConfirmEmail, {email})`

Reply:
:+1: